tests/config_test.go

```diff
@@ -500,13 +500,13 @@ var _ = Describe("[rfe_id:899][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 					&expect.BExp{R: "#"},
```

Comment:
those should be \\# according Jed, maybe worth to handle on other PR?

Reply:
Yes, the string in `expect.BExp{R: str}` is actually a regular expression. Most of our tests don't keep it in mind. I make sure that at least the code that was added in this PR is aware to it. Thanks for the comment.

Reply:
`\\#` correction of course